Subject: [PATCH] wip:refactoring files
---
Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	(revision bcd192063f4a80cc85c94bc43f81ecd7211d5c97)
+++ b/main.py	(revision 3f160f36cfdd19dc2109951217d0eb2ce9e80595)
@@ -1,16 +1,5 @@
-users: list[dict] = [
-    {"name":"Kacper", "surname":"Macioch", "posts": 999 },
-    {"name":"Michał", "surname":"Lębryk", "posts": 979 },
-    {"name":"Dominik", "surname":"Kużnik", "posts": 989 },
-    {"name":"Leon", "surname":"Hajdus", "posts": 229 }
-
-]
-
-
-def read_users(user_list: list[dict]) -> None:
-    for user in user_list:
-        print(f"Twój znajomy {user['name']}: {user['surname']}, opublikował{user['posts']} postów")
-
+from models.data import users
+from utils.crud import read_users
 
 
 if __name__ == "__main__":
Index: models/data.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/models/data.py b/models/data.py
new file mode 100644
--- /dev/null	(revision 3f160f36cfdd19dc2109951217d0eb2ce9e80595)
+++ b/models/data.py	(revision 3f160f36cfdd19dc2109951217d0eb2ce9e80595)
@@ -0,0 +1,5 @@
+users: list[dict] = [
+    {"name":"Kacper", "surname":"Macioch", "posts": 999 },
+    {"name":"Michał", "surname":"Lębryk", "posts": 979 },
+    {"name":"Dominik", "surname":"Kużnik", "posts": 989 },
+    {"name":"Leon", "surname":"Hajdus", "posts": 229 }]
\ No newline at end of file
Index: utils/crud.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/utils/crud.py b/utils/crud.py
new file mode 100644
--- /dev/null	(revision 3f160f36cfdd19dc2109951217d0eb2ce9e80595)
+++ b/utils/crud.py	(revision 3f160f36cfdd19dc2109951217d0eb2ce9e80595)
@@ -0,0 +1,3 @@
+def read_users(user_list: list[dict]) -> None:
+    for user in user_list:
+        print(f"Twój znajomy {user['name']}: {user['surname']}, opublikował{user['posts']} postów")
\ No newline at end of file
diff --git a/utils/__init__.py b/utils/__init__.py
new file mode 100644
